reframe/frontend/statistics.py

```diff
@@ -100,10 +97,24 @@ def json(self):
                     'outputdir': None,
                     'stagedir': None,
                     'job_stdout': None,
-                    'job_stderr': None
+                    'job_stderr': None,
+                    'time_setup': t.duration('setup'),
+                    'time_compile': t.duration('complete'),
+                    'time_run': t.duration('t.duration'),
+                    'time_sanity': t.duration('sanity'),
+                    'time_performance': t.duration('performance'),
+                    'time_total': t.duration('total')
                 }
-                if check.job:
+                partition, environ = (check.current_partition,
+                                      check.current_environ)
```

Comment:
Better write it as follows, since it doesn't fit in single line:

```suggestion
                partition = check.current_partition
                environ = check.current_environ
```